src/odemis/gui/cont/milling.py

```
@@ -101,12 +101,19 @@ def rectangle_pattern_to_shape(canvas,
                         ref_img: model.DataArray,
                         pattern: RectanglePatternParameters,
                         colour: str = "#FFFF00",
-                        name: str = None) -> EditableShape:
-    """Convert a rectangle pattern to a shape"""
-    rect = RectangleOverlay(cnvs=canvas, colour = colour, show_selection_points = False)
+                        name: str = None,
```

Ruff (0.15.17)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)